Docs - Metrics docs layout and info about query/bytes (#11481)
apache/druid: docs/operations/metrics.md

  -->

> Druid can be [configured](../configuration/index.html#enabling-metrics) to emit metrics that are essential for monitoring query execution, ingestion, coordination, and so on.

Reviewer: ```suggestion
You can configure Druid to [emit metrics](../configuration/index.html#enabling-metrics) that are essential for monitoring query execution, ingestion, coordination, and so on.
```

Metrics may have additional dimensions beyond those listed above.

> > Most metric values reset each emission period, as specified in [`druid.monitoring.emissionPeriod`](../configuration/index.html#enabling-metrics).

Reviewer: ```suggestion
> Most metric values reset each emission period, as specified in `druid.monitoring.emissionPeriod`.
```
We've already linked to enabling metrics. no need to re-link every time.

|Metric|Description|Dimensions|Normal Value|
|------|-----------|----------|------------|
|`query/time`|Milliseconds taken to complete a query.|Common: dataSource, type, interval, hasFilters, duration, context, remoteAddress, id. Aggregation Queries: numMetrics, numComplexMetrics. GroupBy: numDimensions. TopN: threshold, dimension.|< 1s|

> |`query/bytes`|The total number of bytes returned in the query response back to the requesting client by the broker.  (Other services report the total bytes for their portion of the query.)|Common: dataSource, type, interval, hasFilters, duration, context, remoteAddress, id. Aggregation Queries: numMetrics, numComplexMetrics. GroupBy: numDimensions. TopN: threshold, dimension.| |

Reviewer: ```suggestion
|`query/bytes`|The total number of bytes returned to the requesting client in the query response from the broker.  Other services report the total bytes for their portion of the query. |Common: `dataSource`, `type`, `interval`, `hasFilters`, `duration`, `context`, `remoteAddress`, `id`. Aggregation Queries: `numMetrics`, `numComplexMetrics`. GroupBy: `numDimensions`. TopN: `threshold`, `dimension`.| |
```
I'm not sure how we handle this across the docs, but these columns/metrics look like code words, so adding code formatting. Remove if unnecessary.
